portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,49 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** Container class for Form data */
+class FormData {
+  private String nickname; //The alias of the commenter
+  private String comment; // The assocaited comment
+  
+  public FormData(String nickname, String comment) {
+    this.nickname = nickname;
+    this.comment = comment;
+  }
+}
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("Hello Sam");
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String nickname = getParameter(request, "nickname", "anon");
+    String comment = getParameter(request, "comment", "");
+    List<FormData> formdata = Arrays.asList(new FormData(nickname,comment));
```

Comment:
nit: use camelCase for the `formdata` variable name (`formData`).

Reply:
Updated in commit e9d80a6